reflex/state.py

```diff
@@ -3387,6 +3417,19 @@ async def set_state(
                 f"`app.state_manager.lock_expiration` (currently {self.lock_expiration}) "
                 "or use `@rx.event(background=True)` decorator for long-running tasks."
             )
+        elif lock_id is not None:
+            time_taken = self.lock_expiration / 1000 - (
+                await self.redis.ttl(self._lock_key(token))
+            )
+            _validate_lock_warning_threshold(
+                self.lock_warning_threshold, self.lock_expiration
+            )
```

Comment:
this seems like something that should happen during `__init__`, not every time we're persisting state